static/js/components/email/EmailCompositionDialog.js

```diff
@@ -78,6 +79,9 @@ export default class EmailCompositionDialog extends React.Component {
   constructor (props: EmailDialogProps) {
     super(props);
     this.state = editorStateFromProps(props);
+    if (R.isNil(this.state.editorState)) {
```

Comment:
so unless I'm missing something we shouldn't need to manually set `this.state.editorState` to be `EditorState.createEmpty()`